tools/compare-values-schema-content.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+"""
+This script is meant to assist in a manual validation that the content of
+schema.yaml covers values.yaml, and vice versa.
+
+FIXME: It would be nice to run this as part of our CI pipeline to report if
+       schema.yaml and values.yaml gets out of sync, but first we need to
+       address what it means to be out of sync.
+
+       Consider if schema.yaml describes extraLabels, and we in this helm chart
+       have an extra label set in values, how should our comparison realize that
+       its nothing to bother about?
+
+       That kind of complexity is currently an issue for labels, resources,
+       containerSecurityContext, readiness- and livenessProbe's, and hub.config.
+"""
+
+import jsonschema
+import os
+import sys
+
+from collections.abc import MutableMapping
+
+import yaml
```

Comment:
Don't we use ruamel.yaml elsewhere, instead of pyyaml?

Reply:
Our dev-dependencies have pyyaml while I see ruamel.yaml in the images/hub dependencies script.

Do you favor one or the other? I currently don't favor one or the other based on not being aware of differences.

Reply:
Ah, I see. I generally favor ruamel.yaml - it has an 'rt' mode that can write out YAML formatted the same way it is read (including comments!). So chartpress, etc use it. Not critical though.

Reply:
Ah thats very imporitant for chartpress for examole, good to know!

Reply:
Lets switch if that ever cause trouble to not have!